apps/etterlatte-pdltjenester/src/main/kotlin/pdl/mapper/PersonMapper.kt

```diff
@@ -189,6 +191,32 @@ object PersonMapper {
             )
         }
 
+    fun mapPersonNavnFoedselsDato(
+        ppsKlient: ParallelleSannheterKlient,
+        ident: String,
+        hentPerson: PdlHentPersonNavnFoedselsdato,
+    ): PersonNavnFoedselsaar =
+        runBlocking {
+            val navn = ppsKlient.avklarNavn(hentPerson.navn)
+
+            val fnr =
+                if (Folkeregisteridentifikator.isValid(ident)) {
+                    ident
+                } else {
+                    ppsKlient.avklarFolkeregisteridentifikator(hentPerson.folkeregisteridentifikator)
+                        .identifikasjonsnummer
+                }
+            val foedsel = ppsKlient.avklarFoedsel(hentPerson.foedsel)
+
+            PersonNavnFoedselsaar(
+                fornavn = navn.fornavn,
+                mellomnavn = navn.mellomnavn,
+                etternavn = navn.etternavn,
+                foedselsnummer = Folkeregisteridentifikator.of(fnr),
+                foedselsaar = foedsel.foedselsaar,
```

Comment:
Du må sende med hele fødselsdatoen for å kunne beregne korrekt alder.

Reply:
Jepp, ikke alltid det finnes tydeligvis heller